test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorResolverTest.cs

```diff
@@ -149,7 +149,7 @@ public static TheoryData ResolveDirectiveDescriptorsInvalidTagHelperPrefixData
                             },
                             new TagHelperDirectiveDescriptor
                             {
-                                DirectiveText = "*Plain*, " + assemblyA,
+                                DirectiveText = "Microsoft.AspNet.Razor.Runtime.TagHelpers.Valid_P*, " + assemblyA,
```

Comment:
There are other tests in `Resolve_ReturnsPrefixedDescriptorsBasedOnDirectiveDescriptors` that cover `___*` scenario. There is no need to cover `*___*` because it is the same as testing `___*`.